.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+###################################
+##### Global Protection Rule ######
+###################################
+# NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
+*                                               @hashgraph/hedera-sdk
```

Comment:
Would it make sense to add `hedera-sdk-cpp-maintainers` and/or `hedera-sdk-cpp-contributors` here as well?